### ISSUE TITLE
[Bug] Other bookmarks can't be moved

### ISSUE THREAD

**Issue by oleedd** [rpkamp/chrome-minimal-bookmarks-tree#93]

**Comment by oleedd** [comment]
But dragging from "Other bookmarks" works.

**Comment by oleedd** [comment]
Rearranging doesn't work for "Other bookmarks". Seems that dragula is not connected or incorrectly connected to them.

**Comment by rpkamp** [comment]
Why are you seeing "Other bookmarks"? It should not be visible by default.

Which browser are you using?

**Comment by oleedd** [comment]
I don't see the folder. I talk about other bookmarks. They can't be moved.

**Comment by rpkamp** [comment]
No problem here. Which version of the extension are you using and which browser are you running?

**Comment by oleedd** [comment]
The last version. Chrome.
I can move a bookmark from "Other bookmarks" to "Bookmark bar". But not contrariwise.
Dragula doesn't work for the "Other bookmarks" list. But works for "Bookmark bar".

**Comment by oleedd** [comment]
The last version. Chrome.
I can move a bookmark from "Other bookmarks" to "Bookmark bar". But not contrariwise.
Dragula doesn't work inside the "Other bookmarks" list. But works for "Bookmark bar".

**Comment by rpkamp** [comment]
If you can provide some more debugging information, like building the extension locally and installing it in chrome in development mode and then inspecting it, that would be great.

I can't reproduce the error here, and it's impossible to fix something that's not broken (at least not on my machine).

**Comment by oleedd** [comment]
It seems that you incorrectly try to reproduce because the same in several another Chrome versions and Opera. To reproduce need:
\- drop all your bookmarks to "Other bookmarks" (in Chrome bookmarks manager);
\- try to drag&drop some bookmark several positions higher/lower of its actual place to change the order.
So you can drag it but after that it will return to its old position place. Reordering doesn't work for other bookmarks.

**Comment by oleedd** [comment]
It seems that you try to reproduce incorrectly because the same in several another Chrome versions and Opera. To reproduce need:
- drop all your bookmarks to "Other bookmarks" (in Chrome bookmarks manager);
- try to drag&drop some bookmark several positions higher/lower of its actual place to change the order.
So you can drag it but after that it will return to its old position place. Reordering doesn't work for other bookmarks.

**Comment by oleedd** [comment]
It seems that you try to reproduce incorrectly because the same in several another Chrome versions and Opera. To reproduce need:
\- drop all your bookmarks to "Other bookmarks" (in Chrome bookmarks manager);
\- try to drag&drop some bookmark several positions higher/lower of its actual place to change the order.
So you can drag it but after that it will return to its old position place. Reordering doesn't work for other bookmarks.

**Comment by oleedd** [comment]
Not possible to change the place of other bookmark. Do you reproduce it now?

**Comment by rpkamp** [comment]
I've moved all bookmarks from "Bookmarks bar" to "Other bookmarks" and still cannot reproduce it.

Can you tell me which settings you've configured in the extension?

**Comment by oleedd** [comment]
![image](https://user-images.githubusercontent.com/36125949/59969552-70953c80-9558-11e9-8a7d-11314770a585.png)

**Comment by rpkamp** [comment]
Are those settings really blank? How did you manage to do that?
Also, I'm missing quite a few settings. Could you please post _all_ of them?

**Comment by oleedd** [comment]
`<select>`s are blank after installing. So if I don't need some options - I don't configure them and leave them blank.

![image](https://user-images.githubusercontent.com/36125949/59973201-ef17cb80-95a4-11e9-8182-ff69b66c7baa.png)

**Comment by rpkamp** [comment]
It still works for me with those settings :man_shrugging: 

The only way I can think of to get to the bottom of this is if you install the extension locally in developer mode (so clone this github repository, build the CSS/JS/etc and load it in chrome://extensions) and then right click on the extension "Inspect pop-up" and then report any errors you see in the javascript console.

That option is only available when the extension is installed in developer mode.

**Comment by oleedd** [comment]
I have already built the extension locally before. And the same problem there but no errors in the console.
> That option is only available when the extension is installed in developer mode.

For me the right click menus are the same and both have "Inspect pop-up". So no need to build to check.

**Comment by rpkamp** [comment]
> And the same problem there but no errors in the console.

Then I have absolutely _no_ idea how to fix this :man_shrugging: 

> For me the right click menus are the same and both have "Inspect pop-up". So no need to build to check.

Right, for me as well. I could have sworn it didn't show in the past when you installed an extension from the chrome store.

**Comment by oleedd** [comment]
I am not sure that you understand the issue correctly so I have recorded a video which displays the problem: [Video.zip](https://github.com/rpkamp/chrome-minimal-bookmarks-tree/files/3317767/Video.zip)
So dragula doesn't work as expected.

**Comment by rpkamp** [comment]
Ah! So the issue is: When I have bookmarks *directly in the root of* "Other bookmarks" then I cannot sort these bookmarks! _That_ I can reproduce.

I don't have bookmarks in the root of the "Other bookmarks" folders; I only have subfolders in "Other bookmarks".

**Comment by oleedd** [comment]
But I haven't talked about any subfolders.
Ha-ha. I wonder that you could understand the issue only after 9 comments of explaining. I almost lost hope)))

**Comment by rpkamp** [comment]
I've created a new issue with the complete description so I don't have go through the entire history here next time I see :wink: 

See https://github.com/rpkamp/chrome-minimal-bookmarks-tree/issues/101
